### ISSUE TITLE
Systematically Use Constant Strings in Different Modules

### ISSUE THREAD

**Issue by gkapfham** [GatorEducator/gatorgrouper#113]
The source code for different modules contains hard-coded strings! For instance, the main `gatorgrouper.py` file contains the following source code:

```
# generate the groups and display them
    if GG_ARGUMENTS.grouping_method == "rrobin" and GG_ARGUMENTS.num_group is DEFAULT_NUMGRP:
        GROUPED_STUDENT_IDENTIFIERS = group_rrobin_group_size(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.group_size)
    elif GG_ARGUMENTS.grouping_method == "rrobin" and GG_ARGUMENTS.num_group is not DEFAULT_NUMGRP:
        GROUPED_STUDENT_IDENTIFIERS = group_rrobin_num_group(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group)
    elif GG_ARGUMENTS.num_group is DEFAULT_NUMGRP:  # default to random method
        GROUPED_STUDENT_IDENTIFIERS = group_random_group_size(
            SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.group_size)
    else:
        GROUPED_STUDENT_IDENTIFIERS = group_random_num_group(
SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group)
```

I suggest that we start to avoid using hard-coded strings like `rrobin` and instead use constant variables. Also, I suggest that the names of the constant variables be much more descriptive.

**Comment by baldeosinghm** [comment]
Mo, Lancaster, and I will start working on refactoring the code

**Comment by baldeosinghm** [comment]
Working on changes to remove hard coding in `quickfix/constant-strings`

**Comment by baldeosinghm** [comment]
Fixed this issue with pull request  73ee8bef528f9ff358effea473f268e81e0b3968
